wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/server/WorkspaceManager.java

```diff
@@ -160,13 +172,14 @@ public UsersWorkspaceImpl createWorkspace(WorkspaceConfig config, String owner,
      * @throws ServerException
      *         when any server error occurs
      */
-    public UsersWorkspaceImpl getWorkspace(String workspaceId) throws NotFoundException, ServerException, BadRequestException {
-        requiredNotNull(workspaceId, "Required non-null workspace id");
-        return normalizeState(workspaceDao.get(workspaceId));
+    public UsersWorkspaceImpl getWorkspace(String key) throws NotFoundException, ServerException, BadRequestException {
+        requiredNotNull(key, "Required non-null workspace key");
+        return normalizeState(getByKey(key));
     }
 
+
     /**
-     * Gets workspace by name and owner.
+     * Gets workspace by name and owner. Deprecated. Use getWorkspace(String key) intstead.
```

Comment:
It would be better to use @deprecated and @link for this information
https://github.com/codenvy/codenvy/blob/master/plugins/plugin-bitbucket/codenvy-plugin-bitbucket-ext-bitbucket/src/main/java/org/eclipse/che/ide/ext/bitbucket/client/BitbucketClientService.java#L156-156